spring-batch-samples/src/main/java/org/springframework/batch/sample/metrics/README.md

```diff
@@ -0,0 +1,32 @@
+## About this sample
```

Comment:
If we're starting to gets docs here and docs there, it may be time to add a /docs directory and start to collect them. Failing that, I'd go with a single readme and collect all the information in that one place (until it is too big for that, which then justifies having a /docs directory and the beginnings of a manual).

Reply:
I removed that additional readme file and moved its content to the root readme.